docs/iris/src/whatsnew/contributions_3.0.0/incompatiblechange_2020-May-15_change_default_unit_loading.txt

```diff
@@ -0,0 +1 @@
+* NetCDF files containing variables without units will be given a unit of "unknown" rather than "1"
```

Comment:
This needs fleshing out to make it clearer how it may affect the user.
Something like ... 
"When loading data from netcdf-CF files, if a variable has no "units" property the corresponding Iris object will now have a ".units='unknown'".  Previously, these cases defaulted to "units=1".  
This applies to cubes, coordinates, and ancilary data."